tests/MongoDB.Analyzer.Tests.Common.TestCases/Builders/BuildersBasic.cs

```diff
@@ -28,6 +30,41 @@ public void Filters_single_expression_exists()
             _ = Builders<User>.Filter.Exists(u => u.Address, false);
         }
 
+
+        [BuildersMQL("{ \"Address\" : { \"$exists\" : true } }", 41)]
+        [BuildersMQL("{ \"Address\" : { \"$exists\" : false } }", 42, 42)]
+        [BuildersMQL("{ \"Address\" : { \"$exists\" : false } }", 43)]
+        [BuildersMQL("{ \"Address\" : { \"$exists\" : true } }", 44, 44)]
+        [BuildersMQL("{ \"Age\" : { \"$exists\" : true } }", 45)]
+        [BuildersMQL("{ \"Address\" : { \"$exists\" : false } }", 46, 46)]
+        public void Filters_single_expression_exists_with_Variable()
+        {
+            var x = Builders<User>.Filter.Exists(u => u.Address, true);
+            x = Builders<User>.Filter.Exists(u => u.Address, false);
+            var y = Builders<User>.Filter.Exists(u => u.Address, false);
```

Comment:
We probably don't need those test as part of this PR.
For this PR just combine `Sort_single_expression_descending` and  `Sort_single_expression_ascending` to single `Sort_single_expression` test.

Reply:
Done